libs/ui/src/ui_strings/app_strings.tsx

```diff
@@ -359,7 +359,7 @@ export const appStrings = {
   instructionsBmdLoadPaper: () => (
     <UiString uiStringKey="instructionsBmdLoadPaper">
       Please feed one sheet of paper into the front input tray. The printer will
-      automatically grab the paper when positioned correctly.
```

Comment:
Somehow this felt a little aggressive

Reply:
We could remove the second sentence all together, it doesn't feel that necessary to me